presidio-analyzer/analyzer/analyzer_engine.py

```diff
@@ -28,23 +28,30 @@ def __init__(self, registry=None, nlp_engine=None):
         self.registry = registry
         # load all recognizers
         registry.load_predefined_recognizers()
+        self.app_tracer = app_tracer
 
     # pylint: disable=unused-argument
     def Apply(self, request, context):
-        logging.info("Starting Apply")
+        logger.info("Starting Analyzer's Apply")
+
         entities = AnalyzerEngine.__convert_fields_to_entities(
             request.analyzeTemplate.fields)
         language = AnalyzerEngine.get_language_from_request(request)
-        results = self.analyze(request.text, entities, language,
+
+        correlation_id = str(uuid.uuid4())
```

Comment:
```suggestion
       #correlation is used to group all traces related to on request
        correlation_id = str(uuid.uuid4())
```

Or something like that